Makefile

```diff
@@ -22,7 +22,7 @@ docker-preview: docker-build
 	docker run -it -v $(shell pwd):/chef-web-docs \
 		-w /chef-web-docs/public \
 		-p 8000:8000 chefes/buildkite \
-		bash -c 'export PATH=$$PATH:/chef-web-docs/doctools; python -m SimpleHTTPServer'
+		bash -c 'export PATH=$$PATH:/chef-web-docs/doctools; python -m http.server'
```

Comment:
what was this change for? is this a consequence of a python / library version change?

Reply:
Yes, `python -m SimpleHTTPServer` runs well until Python-3. But in docker container `Python 3.7.0` was getting installed which `uses http.server module`. Therefore, when I changed the command to `python -m http.server`, I was able to see the output.